client-tools/Shell/ClientFeature.cpp

```diff
@@ -166,6 +166,19 @@ arangosh without connecting to a server.)");
         new StringParameter(&_jwtSecretFile),
         arangodb::options::makeDefaultFlags(
             arangodb::options::Flags::Uncommon));
+
+    options->addOption(
+        "--server.jwt-token",
+        "If enabled, the JWT token is used directly for authentication. You "
+        "can either "
+        "specify the token directly or use the - sign to be prompted for it to "
```

Comment:
I had to read this 3 times to understand, maybe change it to something like:

```suggestion
        "specify the token directly or set the value to `-` to get prompted for the token to "
```